integration_tests/data/data_coalesce_fields.csv

```diff
@@ -1,4 +1,4 @@
-id,numeral,numeral__st,numeral__bl,version__string,version__double,__red_herring
-1,1,,,pi,,not
-2,,one,,,3.14,this
-3,,,TRUE,,,one
+id,numeral,numeral__st,numeral__bl,version__string,version__double,properties__numeral__st,properties__numeral__bl,__red_herring
+1,1,,,pi,,1,not
+2,,one,,,3.14,one,this
+3,,,TRUE,,,,,one
```

Comment:
I just ran and failed integration tests locally. I think these columns may just be slightly misaligned.
```suggestion
id,numeral,numeral__st,numeral__bl,version__string,version__double,properties__numeral__st,properties__numeral__it,properties__numeral__bl,__red_herring
1,1,,,pi,,,1,,not
2,,one,,,3.14,one,,,this
3,,,TRUE,,,,,true,one
```